composer.json

```diff
@@ -55,7 +55,8 @@
         "knplabs/knp-menu-bundle": "^2.1",
         "ezyang/htmlpurifier": "^4.6",
         "nelmio/cors-bundle": "^1.4",
-        "cebe/markdown": "^1.1"
+        "cebe/markdown": "^1.1",
+        "algolia/algoliasearch-client-php": "^1.18"
```

Comment:
Yes I didn't commit it because I wll probably have to rebase a few times. I will add it to the correct commit when I polish the PR 👌